yt_dlp/extractor/arte.py

```diff
@@ -135,6 +135,7 @@ def _real_extract(self, url):
                 'Video is not available in this language edition of Arte or broadcast rights expired', expected=True)
 
         formats, subtitles = [], {}
+        potential_duplicates = []
```

Comment:
The var name isnt clear imo. potential duplicate what? `potential_duplicate_formats` works. Or maybe `secondary_formats`?

Reply:
`secondary_formats` seems better, that way we get `formats.extend(secondary_formats)`